src/main/java/reposense/model/reportconfig/ReportBranchData.java

```diff
@@ -60,6 +98,14 @@ public Long getFileSizeLimit() {
         return fileSizeLimit;
     }
 
+    public LocalDateTime getSinceDate() {
+        return sinceDate;
+    }
+
+    public LocalDateTime getUntilDate() {
+        return untilDate;
+    }
+
     @Override
     public boolean equals(Object obj) {
```

Comment:
Please add the since and until dates into the `equals` method and fix the according tests.